ailab/README.md

```diff
@@ -0,0 +1,34 @@
+# Launch Benchmark Remotely
+
+You can set up a server and and launch the benchmark run remotely.
+
+## Server Setup
+
+#### Install DJango
+```
+pip install Django
+```
+
+#### Config MySQL Database
+- Install MySQL on the server
+- Create a MySQL database
+- In the current directory (ailab), run
```

Comment:
run what?

Reply:
Should swap this line with line 16. Will fix it in the next commit.